### PR TITLE
fix: fixed the dead link related issue

### PR DESCRIPTION
**Pull request by anikdhabal**
Test is failing due to those dead links.

### CODE REVIEW

**Comment by riderx** [comment]
Thanks for the PR,
The link should be modified in the plugin repository not here, here it's a script show fetch them :)

File: src/content/plugins-tutorials/home-indicator.md [hunk 1]
```diff
@@ -59,6 +59,6 @@ getWindow().setDecorFitsSystemWindows(false);
 
 ## Credits
 
-This plugin was created originally for [Kick.com](https://kick.com) by [Capgo](https://capgo.app)
+This plugin was created originally for [Kick.com](https://g.co/kgs/6Nvu3n) by [Capgo](https://capgo.app)
```
Comment (riderx) [inline src/content/plugins-tutorials/home-indicator.md:62]: i'm not sure why this modif ?

File: src/content/plugins-tutorials/native-audio.md [hunk 2]
```diff
@@ -53,6 +53,6 @@ NativeAudio.play({
 
 The `preload` method is used to load an audio file into memory, and the `play` method is used to play the loaded audio file.
 
-Other supported methods include `pause`, `resume`, `loop`, `stop`, `unload`, `setVolume`, `getDuration`, and `getCurrentTime`. You can refer to the [official documentation](https://github.com/riderx/native-audio/blob/master/README.md) for more details on these methods.
+Other supported methods include `pause`, `resume`, `loop`, `stop`, `unload`, `setVolume`, `getDuration`, and `getCurrentTime`. You can refer to the [official documentation](https://github.com/riderx/) for more details on these methods.
```
Comment (riderx) [inline src/content/plugins-tutorials/native-audio.md:56]: same here, why ?

Comment (anikdhabal) [reply src/content/plugins-tutorials/native-audio.md:56]: This link is really dead.
Screenshot 2023-09-15 074622

File: src/content/plugins-tutorials/home-indicator.md (hunk 1 above)
Comment (anikdhabal) [reply src/content/plugins-tutorials/home-indicator.md:62]: Yeah, this shouldn't modify. Thanks for your review.

**Comment by anikdhabal** [comment]
> Thanks for the PR, The link should be modified in the plugin repository not here, here it's a script show fetch them :)

Ahh , Okay🙏

**Comment by riderx** [comment]
@anikdhabal did you made Pr to those repo then ?

File: src/content/plugins-tutorials/native-audio.md (hunk 2 above)
Comment (riderx) [reply src/content/plugins-tutorials/native-audio.md:56]: good link: https://github.com/Cap-go/native-audio/blob/main/README.md

Comment (anikdhabal) [reply src/content/plugins-tutorials/native-audio.md:56]: Okk

**Comment by anikdhabal** [comment]
> @anikdhabal did you made Pr to those repo then ?

Not yet, but going to made.

**Comment by riderx** [comment]
@anikdhabal can you link them here to be sure I don’t miss one

**Comment by anikdhabal** [comment]
> @anikdhabal can you link them here to be sure I don’t miss one

Ok

**Comment by riderx** [comment]
hey man i'm willing to tip you for that too, don't forget it :)

**Comment by anikdhabal** [comment]
> hey man i'm willing to tip you for that too, don't forget it :)

I forgot to fix this issue. Le me fix that.

**Comment by anikdhabal** [comment]
@riderx Can you provide a valid link of this -https://play.google.com/store/apps/details?id=ee.forgr.capacitor_go

**Comment by riderx** [comment]
i think this is the valid link just i didn't updated the app with latest SDK so my app got removed ^^ i will check that now

**Comment by anikdhabal** [comment]
> i think this is the valid link just i didn't updated the app with latest SDK so my app got removed ^^ i will check that now

Sure

**Comment by riderx** [comment]
it works again :)

**Comment by anikdhabal** [comment]
> it works again :)

Thank you. Now all the broken links are fixed.

**Comment by anikdhabal** [comment]
> it works again :)

Thank you @riderx . Now all the broken links are updated.

**Pull request merged**